### DISCUSSION TITLE
Tailwind UI - Figma

### DISCUSSION THREAD

**Discussion by hylickipiotr** [tailwindlabs/tailwindcss#3302] [General]
@adamwathan
Is the Figma project available when I buy Tailwind UI Application UI components licence?

**Comment by adamwathan** [comment]
We have a preview release available in the Tailwind UI Discord and the full release is coming this month 👍🏻

**Reply by hylickipiotr** [reply]
Thank 👍

**Reply by adamwathan** [reply]
Not released yet, very close but we still have about 20-30 components to re-create in Figma before we are fully at parity with what's available as HTML. Within the next 2 weeks for sure.

**Reply by zain** [reply]
is it out yet?

edit: thanks adam! for everyone else, i found it here: https://tailwindui.com/download-figma

**Reply by adamwathan** [reply]
Yep it's out 👍🏻

**Comment by Bingham552** [comment]
I want to encourage that you continue your great work, have a nice day!

[walgreenslistens](https://www.walgreenslistens.biz/)
